### DISCUSSION TITLE
Re-add Supervisor management (/hassio/system) to settings menu

### DISCUSSION THREAD

**Discussion by bcutter** [home-assistant/frontend#13519] [Configuration] [answered]
During the "streamlining" in 2022.5 the icon/link in configuration menu to /hassio/system has been removed.

Now you have various places to
- restart HA
- restart the host

but no place to reload or restart the supervisor.

"/hassio/system" is still there and offers ONE place to manage all system components (HA Core, Supervisor, Host), but not reachable from the default configuration menu.

No one knows if that has been removed by decision or accidentally. People [searched their back off](https://community.home-assistant.io/t/2022-5-streamlining-settings/418477/633) for managing the supervisor without using the CLI.

Please re-add that link in the UI. Preferably under System.

![image](https://user-images.githubusercontent.com/13799156/187400044-1d39ce33-964e-4b69-a012-049f90040dff.jpeg)

**Comment by matthiasdebaat** [comment, marked as answer]
> During the "streamlining" in 2022.5 the icon/link in configuration menu to /hassio/system has been removed. Now you have various places to
> - restart HA
> - restart the host
> but no place to reload or restart the supervisor.

To understand your issue, can you elaborate more about why you want this page back? When do you want to restart supervisor? Why do you want to reboot your host? How often do you want to restart everything at the same time?

> No one knows if that has been removed by decision or accidentally. People [searched there back off](https://community.home-assistant.io/t/2022-5-streamlining-settings/418477/633) for managing the supervisor without using the CLI.

The release notes you refer to explain it, and people explained it multiple times again in the forum topic. Most of the people know, I'm not going to repeat that.

> "/hassio/system" is still there and offers ONE place to manage all system components (HA Core, Supervisor, Host), but not reachable from the default configuration menu.
> 
> Please re-add that link in the UI. Preferably under System.

We've changed it with a reason, this page isn't coming back for all users. The URL needs to stay working for older HA versions, so you can keep using it for now.

**Reply by bcutter** [reply]
> To understand your issue, can you elaborate more about why you want this page back? 

To say it with https://community.home-assistant.io/t/2022-5-streamlining-settings/418477/645

> It was a perfect overview with all options, which are now split around at several (mostly empty) places or hidden menus or even not available anymore in general. For me in my personal view, the opposite of streamlining.

...and https://community.home-assistant.io/t/2022-5-streamlining-settings/418477/637 where current options are located:

> Settings → System → Restart button: Restarts Home Assistant

> Developer Tools → YAML → Restart button: Restarts Home Assistant

> Settings → System → Hardware → Overflow Menu (three dots): reboots the host machine, you should hardly ever need to do this, which is why it is less accessible.

Before: everything at one place.
Today: most parts widely spread over several places, supervisor management completely hidden.

Why NOT having one place to control everything?

---------

My personal motivation was to restart the supervisor because of (a recommendation in) https://github.com/thomasmauerer/hassio-addons/issues/93 / https://github.com/hassio-addons/addon-influxdb/issues/251. Was searching for half an hour which was so frustrating.

**Reply by matthiasdebaat** [reply]
This sounds like an Add-on/Supervisor problem. Open a Supervisor issue about missing content in the backup. We shouldn't fix this issue in the UI.

**Reply by bcutter** [reply]
1. This was just an (my) example. One can of course create a custom view with a link to /hassio/system or use the CLI command or create a command line sensor for restarting it or or or - but all that's way more complicated compared to have it in the UI. Removing things adds more focus and clearness. Not in this case as stated with the various examples in my last post.

2. Maybe you wanna point me to the right direction: where is that supervisor repo. Created an issue in the samba backup addon --> redirected to/created another one in the addons repo (no response)... is another issue really helpful?
